package.json

```diff
@@ -42,6 +42,7 @@
     "gulp-insert": "0.5.0",
     "gulp-less": "3.1.0",
     "gulp-load-plugins": "1.3.0",
+    "gulp-mocha": "^3.0.1",
```

Comment:
heck no we deleted our shrinkwrap a while back and haven't looked back. in my experience, npm shrinkwrap only causes headaches.

we'll move to yarn as soon as we can.